packages/auth/src/lib/shared/auth.interface.ts

```diff
@@ -102,7 +103,7 @@ export interface User extends BaseUser {
   sourceId?: string;
   locale?: string;
   isExpired?: boolean;
-  admin?: boolean;
+  isAdmin?: boolean;
```

Comment:
Potentiellement un breaking changes?

Reply:
Peut-etre, mais .admin n'est pas utilisé .... isAdmin l'est déja !

Reply:
Je ne connais pas beaucoup l'API donc je ne suis pas en mesure de répondre a ta question mais est-ce qu'on pourrait conserver les deux et déprécié le `admin`? Donc on aurait une double vérification pour déterminer s'il est admin! On pourrait se laisser une note de supprimer dans 1 an ça nous laisserait le temps d'agir si on en a de besoin.

Reply:
@mbarbeau Pourrais-tu répondre a la question de @pelord?

Reply:
on va mettre admin deprecated + ajouter isadmin

Reply:
@pelord a raison, c'est 'isAdmin' qui est utilisé

Reply:
Let's go for it